test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/AnonymizedExportUsingAcrTests.cs

```diff
@@ -0,0 +1,479 @@
+            HttpResponseMessage response = await WaitForCompleteAsync(contentLocation);
+            string responseContent = await response.Content.ReadAsStringAsync();
+
+            Assert.Equal(HttpStatusCode.BadRequest, response.StatusCode);
+            Assert.Contains("Anonymization configuration 'not-exist.json' not found.", responseContent);
+
+            // Only check metric for local tests
+            if (_isUsingInProcTestServer)
+            {
+                Assert.Single(_metricHandler.NotificationMapping[typeof(ExportTaskMetricsNotification)]);
+            }
+        }
+
+        [SkippableFact]
+        public async Task GivenALargeConfiguration_WhenExportingAnonymizedData_ThenBadRequestShouldBeReturned()
+        {
+            var registry = GetTestContainerRegistryInfo();
+
+            // Here we skip local E2E test since we need Managed Identity for container registry token.
+            // We also skip the case when environmental variable is not provided (not able to upload configurations)
+            Skip.If(_isUsingInProcTestServer || registry == null);
+            string largeConfig = new string('*', (1024 * 1024) + 1); // Large config > 1MB
+            await PushConfigurationAsync(registry, TestRepositoryName, TestRepositoryTag, largeConfig);
+
+            _metricHandler?.ResetCount();
+            var dateTime = DateTimeOffset.UtcNow;
+
+            string containerName = Guid.NewGuid().ToString("N");
+            string reference = $"{registry.Server}/{TestRepositoryName}:{TestRepositoryTag}";
+            Uri contentLocation = await _testFhirClient.AnonymizedExportUsingAcrAsync(TestConfigName, reference, dateTime, containerName);
+            HttpResponseMessage response = await WaitForCompleteAsync(contentLocation);
+            string responseContent = await response.Content.ReadAsStringAsync();
+
+            Assert.Equal(HttpStatusCode.BadRequest, response.StatusCode);
+            Assert.Contains("Anonymization configuration is too large", responseContent);
+
+            // Only check metric for local tests
+            if (_isUsingInProcTestServer)
+            {
+                Assert.Single(_metricHandler.NotificationMapping[typeof(ExportTaskMetricsNotification)]);
+            }
+        }
+
+        private async Task<HttpResponseMessage> WaitForCompleteAsync(Uri contentLocation)
+        {
+            HttpStatusCode resultCode = HttpStatusCode.Accepted;
+            HttpResponseMessage response = null;
+            while (resultCode == HttpStatusCode.Accepted)
+            {
+                await Task.Delay(5000);
+
+                response = await _testFhirClient.CheckExportAsync(contentLocation);
+
+                resultCode = response.StatusCode;
+            }
+
+            return response;
+        }
+
+        private async Task<IList<Uri>> CheckExportStatus(HttpResponseMessage response)
```

Comment:
For CheckExportStatus, DownloadBlobAndParse, and GetCloudStorageAccountHelper is there enough unique requirements for this test class to rewrite them or could the helper methods in ExportTestHelper be used?